subs/pantry/package.yaml

```diff
@@ -62,6 +62,7 @@ dependencies:
 - resourcet
 - rio-prettyprint
 - mtl
+- extra
```

Comment:
I think we've been trying to avoid adding dependencies on these kinds of packages.